doc/nvim-treesitter.txt

```diff
@@ -449,228 +449,236 @@ Note: This is highly experimental, and folding can break on some types of
 ==============================================================================
 HIGHLIGHTS                                        *nvim-treesitter-highlights*
 
+The following is a list of highlights groups, the syntactic elements they
+apply to, and some examples.
+
 							     *hl-TSAnnotation*
 `TSAnnotation`
```

Comment:
But that's TSAttribute now. Can you change the title? The description is ok but it's now `@attribute` `@annotation no longer exists.